apps/volontulo/tests/forms/test_edit_profile.py

```diff
@@ -83,3 +83,14 @@ def test__is_validate__distinct_new_passwords__raise_exception(self):
         )
         form = EditProfileForm(data=form_data)
         self.assertRaises(ValidationError, form.is_valid)
+
+    def test__no_is_avatar_field_on_edit_profile_form(self):
+        u"""Test if Is Avatar field is not visible."""
```

Comment:
Unicode handling refactoring is not a part of this ticket. As well as adding unicode_literals.